sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -1306,7 +1306,8 @@ pi_result resetCommandLists(pi_queue Queue) {
 pi_result _pi_context::getAvailableCommandList(
     pi_queue Queue, pi_command_list_ptr_t &CommandList, bool UseCopyEngine,
     bool AllowBatching, ze_command_queue_handle_t *ForcedCmdQueue) {
-    
+
+#if SYCL_EXT_ONEAPI_GRAPH
```

Comment:
I guess even if we set this macro in `sycl/include/sycl/ext/oneapi/experimental/graph.hpp`, it wouldn't get
set picked up here as the `graph.hpp` file isn't included. What's the easiest way to flip all these `SYCL_EXT_ONEAPI_GRAPH` macros to true during development?

Reply:
Thanks, I think the CMake option would be the easiest to use. I added a `SYCL_ENABLE_GRAPH` flag to CMake that sets `SYCL_EXT_ONEAPI_GRAPH` in `include/sycl/feature_test.hpp.in`. There is also the new `--enable-sycl-graph` for `buildbot/configure.py`.

Reply:
Nice one, looks good 👍